nengo_dl/__init__.py

```diff
@@ -32,6 +32,10 @@
 
 os.environ["TF_CPP_MIN_LOG_LEVEL"] = "1"
 tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.WARN)
+
+# disable eager execution (for now, pending full TF 2.0 compatibility)
+tf.compat.v1.disable_eager_execution()
```

Comment:
Why for now? Would we ever want eager execution?

Reply:
We wouldn't use it internally in NengoDL, but it'd be nice if NengoDL still worked when eager execution was enabled (so users could use eager execution in the same script they're doing NengoDL stuff, if they wanted).  Which might just mean disabling eager execution before any of our build functions and then re-enabling it afterwards?  I didn't start looking yet into whether there's a clean way to do it.

Reply:
That made me think though that we should probably mention this in the changelog, since it might have unexpected effects for users.  Added a fixup for that.